Source/EasyNetQ/ConnectionConfigurationExtensions.cs

```diff
@@ -41,34 +41,51 @@ public static void SetDefaultProperties(this ConnectionConfiguration configurati
             {
             }
 
-        var hostname = Environment.MachineName;
-
-        var netVersion = Environment.Version.ToString();
-        configuration.Product ??= applicationName;
-        configuration.Platform ??= hostname;
-        configuration.Name ??= applicationName;
```

Comment:
Why those 3 properties were set? I think it was superfluous if the goal here was only fill `configuration.ClientProperties`.